gradio/blocks.py

```diff
@@ -1827,6 +1830,12 @@ def reverse(text):
         self.config = self.get_config_file()
         self.max_threads = max_threads
         self._queue.max_thread_count = max_threads
+        self._queue.default_concurrency_limit = int(
```

Comment:
can we call self._queue.set_concurrency_limit from here instead? Feels like this logic should be handled by the queue

Reply:
Oh whoops that's what I meant to do, will fix